SPECS/packer/packer.spec

```diff
@@ -1,13 +1,13 @@
 Summary:        Tool for creating identical machine images for multiple platforms from a single source configuration.
 Name:           packer
-Version:        1.9.4
-Release:        1%{?dist}
```

Comment:
Please double check the license.  This is probably ok, but several hashicorp products are switching to non-open-source licenses.  Approved otherwise

Reply:
That's a good catch - they have switched to the Business Source License, which seems restrictive enough that we may be unable to use this newer version. I'll hold off on this PR until we figure this out.

Reply:
Moved back 1.9.5 where they license has still been Mozilla Public License 2.0.